src/AchievementWindow.cpp

```diff
@@ -0,0 +1,95 @@
+#include "AchievementWindow.h"
```

Comment:
It's nice to have the system headers and app headers grouped together (alpha-sorted).